lib/deck.rb

```diff
@@ -0,0 +1,19 @@
+require 'pry'
+
+class Deck 
+    attr_accessor :cards
+
+    def initialize(cards); @cards = cards end
+
+    def count; @cards.length end
```

Comment:
While this single line method is cute, however, I would recommend using the more verbose syntax instead.

Stylistic choices are often personal preference so you can take this advice with a grain of salt.